src/scripts/main.js

```diff
@@ -1,3 +1,53 @@
 'use strict';
 
-// write code here
+const tbody = document.querySelector('tbody');
+const rows = [...tbody.children];
+
+const columnIndex = {
+  Name: 0,
+  Position: 1,
+  Age: 2,
+  Salary: 3,
+};
+
+document.querySelector('thead').addEventListener('click', e => {
+  switch (e.target.textContent) {
+    case 'Name':
+      sortTable(columnIndex.Name);
+      break;
+    case 'Position':
+      sortTable(columnIndex.Position);
+      break;
+    case 'Age':
+      sortTable(columnIndex.Age);
+      break;
+    case 'Salary':
+      sortTable(columnIndex.Salary);
+      break;
+    default:
+      throw new Error('Unknown step');
+  }
+});
+
+function sortTable(column) {
```

Comment:
It is a good tone to declare funtions before use, so it's better to move this logic above

Reply:
Roger that